README.md

```diff
@@ -38,8 +52,11 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
-```
+There is are mainly two scopes: global and local.
+
+Global scopes refers to variables accessible by any function.  Local refers to variables accessible inside a function.
+
+There is also a closure which is a function that access to a parent scope's variables.
 
```

Comment:
:+1:  How are variables in the global scope defined? 
Also would note: all functions (and only functions) create a new local scope.